eslint.config.mjs

```diff
@@ -314,7 +311,6 @@ export default [
                 lwcRuntimeFlags: true,
                 process: true,
                 LWC: true,
-                spyOnAllFunctions: true,
```

Comment:
This key is covered by `...globals.jasmine`, we don't need to specify it ourselves.